bgpd/bgp_mplsvpn.h

```diff
@@ -124,6 +126,19 @@ static inline int vpn_leak_to_vpn_active(struct bgp *bgp_vrf, afi_t afi,
 		return 0;
 	}
 
+	ifp = if_get_vrf_loopback(bgp_vrf->vrf_id);
+	/* should not happen */
+	if (!ifp) {
+		if (pmsg)
+			*pmsg = "loopback interface not defined";
+		return 0;
+	}
+	if (!if_is_up(ifp)) {
```

Comment:
if (if_is_vrf(ifp) && !if_is_up(ifp))

we do not care of the default lo being down

Reply:
done